challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  return input.map{|x| x.capitalize}
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  return input.select{|x| x=="dog"} 
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  return input.reject{|x| x=="dog"}
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  return input.size
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  return input[0..1]
```

Comment:
`first` is an option that's more commonly used.
https://ruby-doc.org/core-2.5.0/Enumerable.html#method-i-first